pymongo/asynchronous/bulk.py

```diff
@@ -407,7 +509,23 @@ async def _execute_command(
                     if self.ordered and "writeErrors" in result:
                         break
                 else:
-                    to_send = await bwc.execute_unack(cmd, ops, client)
+                    if self.is_encrypted:
```

Comment:
Can this be refactored into a helper method? Something like:
```python
                    result, to_send = await self._execute_batch_unack(cmd, ops, client)
```

We can also use the new helper in execute_op_msg_no_results down below.